src/styles/card.scss

```
@@ -0,0 +1,126 @@
+.card-container {
+  --columnsCount: 1;
+
+  max-width: 944px;
+  margin: auto;
+
+  display: grid;
+  grid-template-columns: repeat(var(--columnsCount), 1fr);
+  justify-content: center;
+  justify-items: center;
+  gap: 46px 48px;
+
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    --columnsCount: 2;
+  }
+
+  @media (min-width: 768px) {
+    --columnsCount: 3;
+  }
+
+  @media (min-width: 1024px) {
+    --columnsCount: 4;
+  }
+}
+
+.card {
+  border: $cardBorderProperties;
+  width: $cardWidth;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  transition: transform ease;
+  transition-duration: $transitionDuration;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    color: $mainTextColor;
+    padding-bottom: 4px;
+    @include titleFont;
+  }
+
+  &__code {
+    padding-bottom: 16px;
+    align-self: baseline;
+    color: $secondaryTextColor;
+    @include secondaryFont;
+  }
+
+  &__evaluation {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 100%;
+    padding-bottom: 24px;
+
+    &-review {
+      text-align: right;
+      color: $mainTextColor;
+      @include secondaryFont;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    padding-bottom: 16px;
+
+    &-title {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondaryTextColor;
+    }
+
+    &-num {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      text-align: right;
+      color: $mainTextColor;
+    }
+  }
+
+  &__btn {
+    width: 166px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 14px;
+    text-align: center;
+    color: $whiteColor;
+    background-color: $blueAccent;
+    padding: 12px 70px;
+    text-transform: uppercase;
+    border-radius: 5px;
+    border: 1px solid $blueAccent;
+    transition: background-color ease;
+    transition-duration: $transitionDuration;
+
+    &:hover {
+      color: $blueAccent;
+      background-color: $whiteColor;
+      border: 1px solid $blueAccent;
```

```suggestion
    border: 1px solid transparent;
    transition: background-color ease, color ease, border-color ease;
    transition-duration: $transitionDuration;

    &:hover {
      color: $blueAccent;
      background-color: $whiteColor;
      border-color: $blueAccent;
```
- The border is the same on `hover`. Maybe it should be transparent by default?
- You apply to all properties that you need. Color and border color too. So the transition will be smooth for all properties